jenkins-pipelines/openshift/Jenkinsfile

```diff
@@ -36,20 +36,21 @@ node ('openstack-jenkins-slave') {
     }
 
     stage('Set variables HEAT deployment') {
-        sh("source ./openstack_rc.sh ; openstack floating ip list | grep 10.2.1.101 | awk '{print \$4}' > env.vars")
-
-        /* TODO: bastionHost isn't used, but does the same as the line above (but better). Fix!! */
-        sh("source ./openstack_rc.sh ; export bastionHost=\$(openstack stack output show openshift bastion_ip -f value -c output_value)")
+        sh("""
+            source ./openstack_rc.sh;
+            openstack stack output show openshift bastion_ip -f value -c output_value > env.vars
+        """)
     }
 
     def bastionip = readFile("env.vars").trim()
 
     stage('Test HEAT deployment') {
+        sh("echo BASTION IP = ${bastionip}")
         sh("ssh -o StrictHostKeyChecking=no -i id_rsa_jenkins cloud-user@$bastionip 'uname -a'")
     }
 
     stage('checkout openshift deployment code') {
-		/* openshift-deplyment-ansible is cloned on the Bastion Host via HEAT, so we don't need to clone it here */
+        /* openshift-deplyment-ansible is cloned on the Bastion Host via HEAT, so we don't need to clone it here */
```

Comment:
Do we really need this comment - it made sense when we'd recently moved to cloning the repo in the heat code, but now it seems unnecessary. Its implied that the code is already there by the fact that we move to the directory it is cloned to.

Reply:
Personally, I think it's helpful, because a Jenkinsfile usually does the "git clone", but here we do something different.

Reply:
OK, well I don't feel that strongly about it to be honest, it was just an observation. If you'd prefer to leave it there for that reason then lets do that.